clients/drcachesim/mirage/fe/gen_opnd_api.cpp

```diff
@@ -0,0 +1,242 @@
+#include "gen_opnd_api.h"
```

Comment:
Is there a more descriptive name than "fe/": I don't know what "fe" means.  "Front end"?  "front_end/" would be better.